reflex/components/__init__.py

```diff
@@ -15,17 +15,6 @@
 from .overlay import *
 from .typography import *
 
-# Add the convenience methods for all the components.
```

Comment:
I think we can actually bring this back now and remove the manually listing of every component since we have the `.pyi` files. But we can bring that in another PR, can we add this comment back for now?

Reply:
Ticket made will uncomment it https://github.com/reflex-dev/reflex/issues/1865#issue-1910493391